client/react-native.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  assets: ['./js/assets/fonts/'],
```

Comment:
If you want to add more fonts to your project ask Kanika since she is the person who is responsible for adding packages and linking dependencies.